### PR TITLE
Update toplist_body.html

### PR DESCRIPTION
**Pull request by toxyy**
changed to twig syntax

### CODE REVIEW

File: styles/prosilver/template/toplist_body.html [hunk 1]
```diff
@@ -1,122 +1,122 @@
...
-		<!-- BEGIN toppostrow -->
-				<tr class="<!-- IF toppostrow.S_ROW_COUNT is even -->bg1<!-- ELSE -->bg2<!-- ENDIF -->">
-					<td><a href="{toppostrow.POST_ID}" class="post">{toppostrow.MINI_POST_IMG}{toppostrow.POST_DATE}&nbsp;{toppostrow.POST_SUBJECT}</a><br />{toppostrow.MESSAGE}</td>
-					<td>{toppostrow.POST_THANKS}</td>
-					<td>{toppostrow.POST_AUTHOR}</td>
-					<td><span style="display: block;">{L_REPUT}{L_COLON}&nbsp;{toppostrow.POST_REPUT}
-						<!-- IF toppostrow.S_THANKS_REPUT_GRAPHIC -->
-						<span style="display: block; width: {toppostrow.THANKS_REPUT_GRAPHIC_WIDTH}; height: {toppostrow.THANKS_REPUT_HEIGHT}; background: url({toppostrow.THANKS_REPUT_IMAGE_BACK}); background-repeat: repeat-x;"><span style="display: block; height: {toppostrow.THANKS_REPUT_HEIGHT}; width: {toppostrow.POST_REPUT}; background: url({toppostrow.THANKS_REPUT_IMAGE}); background-repeat: repeat-x;">&nbsp;</span></span>
-						<!-- ENDIF -->
+		{% for tprow in loops.toppostrow %}
```
Comment (rxu) [inline styles/prosilver/template/toplist_body.html:28]: ```loop``` p[refix is not required for phpBB 3.2. As this extension branch is not BC with the master (for phpBB 3/1)? I think the ```loop``` prefix can be dropped here and after.

Comment (toxyy) [reply styles/prosilver/template/toplist_body.html:28]: I know it isn't required for 3.2, but I wasn't sure if this repository worked for 3.1.x, and if it did I wanted to have it be compatible with both.  If this repo is only for 3.2 then I'll remove all of that.

**Review by toxyy — COMMENTED** [review]
fixed syntax

File: styles/prosilver/template/toplist_body.html (hunk 1 above)
Comment (Galixte) [reply styles/prosilver/template/toplist_body.html:28]: for 3.1.x it’s this branch: https://github.com/rxu/thanks_for_posts/tree/master
for 3.2.x, this branch: https://github.com/rxu/thanks_for_posts/tree/develop-3.2.x

**Comment by CptHindsight** [comment]
Seems all of my work is undermined by the extension that just came out! 

Anyways, I'll change these when I get home. 

In the future, how do I combine all of my changes into one pull request? First time using git. I edited the files one by one and committed them as I did.

File: styles/prosilver/template/toplist_body.html (hunk 1 above)
Comment (toxyy) [reply styles/prosilver/template/toplist_body.html:28]: Alright, I'll change them then. Sorry about all the pull requests, I'm not sure how to combine them into one or do small changes on each file without having to commit. First time using git.

**Comment by rxu** [comment]
You cam do a PR after all you commits are ready, or rather you can modify all files and then commit changes and make PR.

**Comment by toxyy** [comment]
I'll keep that in mind for next time. Every time I edited a file it didn't go on my develop branch, it made its own patch for each edit. I'll probably YouTube it when I'm ready.

**Pull request closed without merging**